packages/annotations/src/format/annotation.js

```diff
@@ -12,8 +12,8 @@ const STORE_KEY = 'core/annotations';
 /**
  * Applies given annotations to the given record.
  *
- * @param {Object} record The record to apply annotations to.
- * @param {Array} annotations The annotation to apply.
+ * @param {Object} record      The record to apply annotations to.
+ * @param {Array}  annotations The annotation to apply.
  * @return {Object} A record with the annotations applied.
```

Comment:
FYI Only: An example where there's no blank line between the last @param and the @return and the @return description is not aligned with the @param descriptions